### PR TITLE
Check Procurement No starts from digit to show the link

### PR DESCRIPTION
**Pull request by aleksievelizhanin**
Add condition to show link to Procurement state registry - only if Procurement No starts with a digit the link will be generated, overwise just a :span with Procurement No. 

Note! may require some addtional styling then there is no link.

### CODE REVIEW

File: app/frontend/src/cljs/teet/contract/contract_common.cljs [hunk 1]
```diff
@@ -12,12 +12,16 @@
...
             [teet.ui.format :as format]
             [teet.util.euro :as euro]
+            [clojure.edn :as edn]
             [teet.user.user-model :as user-model]))
 
 (defn contract-procurement-link
   [{:thk.contract/keys [procurement-number]}]
-  [common/external-contract-link {:href (str (environment/config-value :contract :state-procurement-url) procurement-number)}
-   (str (tr [:contracts :state-procurement-link]) " " procurement-number)])
+  (if (js/Number.isInteger (edn/read-string (subs procurement-number 0 1)))
```
Comment (jkorpelin) [inline app/frontend/src/cljs/teet/contract/contract_common.cljs:20]: Is the edn dependency necessary? Would just `js/parseInt` and then cljs core `integer?` be enough?

**Review by jkorpelin — APPROVED** [review]
Other than the minor comment lgtm

**Comment by jkorpelin** [comment]
Should the link just be not shown if the id is wrong?

**Comment by aleksievelizhanin** [comment]
The AC requiers to show the Procurement No (not as a link) if it is started not from digit, not just hide the link..

**Pull request merged**